cms/models.py

```diff
@@ -51,3 +52,23 @@ class Meta:
 
     def __str__(self):
         return f'{self.post}-{self.caption}'
+
+
+class MenuItem(models.Model):
+    class Meta:
+        verbose_name = 'položka v menu'
+        verbose_name_plural = 'položky v menu'
+        ordering = ['-priority', ]
+
+    caption = models.CharField(
+        verbose_name='názov',
+        help_text='Nápis, ktorý sa zobrazí v menu. Maximálne 2 slová.',
+        max_length=25)
+    url = models.URLField(verbose_name='URL',
+                          help_text='URL stránky kam má preklik viesť')
+    priority = models.SmallIntegerField(
```

Comment:
Ja osobne by som bol za unikatnu prioritu nech je jasne ze tie polozky su zoradene stale a polozky pre site teda tiez lebo su len podmnozinou vsetkych poloziek. A ak by sa jednej polozke zmenila priorita na nejake konkretne cislo, ostatne polozky by sa posunuli.

Akoze je to len navrh ale aj s tym ako to je teraz som vpohode.

Reply:
Môžeme to potom takto nastaviť, len sa mi zatiaľ nechcelo vymýšľať nejaký mechnizmus, ktorý by to naprával v ostatných objektoch

Reply:
druha moznost je nastavit ordering tak, aby to orderoval primarne podla priority a sekundarne podla ID, co by to z toho urobilo viditelne deterministicke poradie